test/python/test_select.py

```diff
@@ -64,3 +64,47 @@ def run_test(sess):
 
         assert (
             self.with_ngraph(run_test) == self.without_ngraph(run_test)).all()
+
+    def test_select_complexshape1(self):
+        a = np.random.randint(2, size=[7])
+        x = np.random.uniform(0, 11, [7, 3, 2, 1])
+
+        p = tf.placeholder(dtype=tf.bool)
+        out = tf.where(p, x, x)
+
+        def run_test(sess):
+            return (sess.run(out, feed_dict={p: a}))
+
+        assert (
+            self.with_ngraph(run_test) == self.without_ngraph(run_test)).all()
+
+    def test_select_complexshape2(self):
+        a = np.random.randint(2, size=[5])
+        x = np.random.uniform(0, 11, [5, 3, 1])
+
+        p = tf.placeholder(dtype=tf.bool)
+        out = tf.where(p, x, x)
+
+        def run_test(sess):
+            return (sess.run(out, feed_dict={p: a}))
+
+        assert (
+            self.with_ngraph(run_test) == self.without_ngraph(run_test)).all()
+
+    #failin test case
+    @pytest.mark.skip(
+        reason=
+        "First dimension of input tensor should match the shape of condition tensor"
```

Comment:
this fails since its not the right input for TF itself, right?

Reply:
true, i added while testing. should we keep or remove?

Reply:
I removed this test.